lib/plugins/blocks.js

```diff
@@ -413,24 +440,26 @@ function inject (bot, { version }) {
   })
 
   bot.updateSign = (block, text) => {
-    const lines = text.split('\n')
+    const lines = [...text.split('\n'), '', '', '']
+      .slice(0, 4)
+      .map(JSON.stringify)
+
     if (lines.length > 4) {
```

Comment:
the above change `.slice(0, 4)` prevents the array from having more than 4 elements, this check no longer does anything